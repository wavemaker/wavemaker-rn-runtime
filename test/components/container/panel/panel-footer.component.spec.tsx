import React, { ReactNode } from 'react';
import { Text, TouchableOpacity } from 'react-native';
import { shallow } from 'enzyme';
import renderer from 'react-test-renderer';
import WmPanelFooter from '@wavemaker/app-rn-runtime/components/container/panel/panel-footer/panel-footer.component';

describe('Test PanelFooter component', () => {
    test('Check validity of sample component', () => {
      const tree = renderer.create(<WmPanelFooter name="test_PanelFooter"/>).toJSON();
      expect(tree).toMatchSnapshot();
    });
});