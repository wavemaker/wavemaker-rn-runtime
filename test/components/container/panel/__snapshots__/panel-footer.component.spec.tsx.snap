// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test PanelFooter component Check validity of sample component 1`] = `
[
  <View
    onLayout={[Function]}
    style={
      {
        "height": 0,
        "width": 0,
      }
    }
  />,
  <View
    style={
      {
        "borderColor": "#dddddd",
        "borderRadius": 6,
        "borderStyle": "solid",
        "borderWidth": 1,
        "marginTop": 8,
        "minHeight": 64,
        "paddingBottom": 8,
        "paddingLeft": 8,
        "paddingRight": 8,
        "paddingTop": 8,
      }
    }
  />,
]
`;
